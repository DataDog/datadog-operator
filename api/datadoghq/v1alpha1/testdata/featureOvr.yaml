apiVersion: datadoghq.com/v1alpha1
kind: DatadogAgent
metadata:
  name: datadog
spec:
  features:
    logCollection:
      enabled: false
      logsConfigContainerCollectAll: true
      containerCollectUsingFiles: false
      containerLogsPath: "/foo/bar"
      podLogsPath: "/baz/dev"
      containerSymlinksPath: "/docker"
      tempStoragePath: "/tmp"
      openFilesLimit: null
  agent:
    log:
      enabled: true
      logsConfigContainerCollectAll: false
      containerCollectUsingFiles: true
      containerLogsPath: "/foo/bar/ignored"
      podLogsPath: "/baz/dev/ignored"
      containerSymlinksPath: "/docker/ignored"
      tempStoragePath: "/tmp/ignore"
      openFilesLimit: 32
