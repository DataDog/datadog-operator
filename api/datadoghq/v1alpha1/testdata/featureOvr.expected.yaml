metadata:
  creationTimestamp: null
  name: datadog
spec:
  features:
    logCollection:
      containerCollectAll: true
      containerCollectUsingFiles: true
      containerLogsPath: /foo/bar
      containerSymlinksPath: /docker
      enabled: true
      openFilesLimit: 32
      podLogsPath: /baz/dev
      tempStoragePath: /tmp
status: {}
